### PR TITLE
fix(TDP-3874): find and group dialog visual glitches

### PR DESCRIPTION
**Pull request by ncomont**
**Link to the JIRA issue**
https://jira.talendforge.org/browse/TDP-3874

**Please check if the PR fulfills these requirements**
- [x] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] The code coverage on new code is > 75 % for backend and > 95% for frontend
- [x] The new code does not introduce new technical issues (sonar / eslint)
- [x] Functional tests have been performed
- [ ] Docker configuration files for config-std or config-cloud profiles are impacted

**Please check the browsers you've tested on**
- [x] Chrome, Firefox, Safari, Edge, IE11
- [ ] No, that's bad, this PR should not be merged !
- [ ] No, and no need to (backend changes only)

**(Optional) What is the current behavior?**
(Additional information to the Jira)

**(Optional) What is the new behavior?**
(Additional information to the Jira)

**(Optional) Other information**:
Also fixed an issue with checkboxes on IE

### CODE REVIEW

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/329/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/329/)

File: dataprep-webapp/src/app/components/transformation/form/cluster/transformation-cluster-params.html [hunk 1]
```diff
@@ -19,12 +19,16 @@
             <thead>
             <tr class="cluster-line">
                 <th>
-                    <label>
-                        <input type="checkbox"
-                               ng-model="clusterParamsCtrl.allCheckboxState"
-                               ng-click="clusterParamsCtrl.refreshClusterState()"
-                               ng-disabled="!!clusterParamsCtrl.isReadonly"/>
-                    </label>
+                    <div class="checkbox">
```
Comment (amaalej) [inline dataprep-webapp/src/app/components/transformation/form/cluster/transformation-cluster-params.html]: You should avoid already taken names: `checkbox` is already an input type.

File: dataprep-webapp/src/app/components/transformation/form/cluster/transformation-cluster-params.html [hunk 2]
```diff
@@ -33,27 +37,32 @@
...
                 ng-repeat="cluster in ::clusterParamsCtrl.details.clusters track by $index">
                 <td>
-                    <label>
-                        <input class="cluster-activation"
-                               ng-model="cluster.active"
-                               type="checkbox"
-                               ng-click="clusterParamsCtrl.refreshToggleCheckbox()"
-                               ng-disabled="!!clusterParamsCtrl.isReadonly"/>
-                    </label>
+                    <div class="checkbox">
```
Comment (amaalej) [inline dataprep-webapp/src/app/components/transformation/form/cluster/transformation-cluster-params.html]: You should avoid already taken names: `checkbox` is already an input type.

File: dataprep-webapp/src/app/components/transformation/form/cluster/transformation-cluster-params.html (hunk 1 above)
Comment (ncomont) [inline dataprep-webapp/src/app/components/transformation/form/cluster/transformation-cluster-params.html]: The `checkbox` class come from Bootstrap : http://getbootstrap.com/css/#forms-example

File: dataprep-webapp/src/app/components/transformation/form/cluster/transformation-cluster-params.html (hunk 2 above)
Comment (ncomont) [inline dataprep-webapp/src/app/components/transformation/form/cluster/transformation-cluster-params.html]: The `checkbox` class come from Bootstrap : http://getbootstrap.com/css/#forms-example

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/338/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/338/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/347/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/347/)

**Pull request merged**